frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -39,7 +39,6 @@ Datasets need the following information added to a single h5ad (AnnData 0.7) for
 - **Dataset-level metadata in uns**:
   - schema_version: `2.0.0`
```

Comment:
todo: update schema version while we're here. Might be worthwhile to do a quick check over any docs that refer to 2.0.0 and update, especially now that we're deprecating a field from 2.0.0

Reply:
actually, maybe we should have a separate documentation ticket--noticing a couple of other spots that could use an update. Maybe this should just link out to the single-cell-curation schema docs? so as to not always have to update two different spots.

either way, I don't want to block this specific ticket with this issue

Reply:
I created a ticket to audit the whole docs: https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/3156